### PR TITLE
Add valid time units for --certificate-ttl flag of kubectl-gs login

### PR DESCRIPTION
**Pull request by ubergesundheit**
Follow up PR for https://github.com/giantswarm/kubectl-gs/pull/565

### CODE REVIEW

**Review by MarcelMue — APPROVED** [review]
LGTM after klubectl-gs is merged

File: src/content/ui-api/kubectl-gs/login.md
Comment (marians) [inline src/content/ui-api/kubectl-gs/login.md:58]:
```suggestion
- `--certificate-ttl` - How long the client certificate should live for. When creating client certificates, we recommend using short expiration periods. Valid time units are "s", "m", "h".
```

**Review by marians — APPROVED** [review]
OK if we remove the nonsensical short units (see my suggestion) ;-)

**Pull request merged**
